AdyenCardScanner/Sources/CardImageParser.swift

```diff
@@ -211,26 +211,26 @@ class CardImageParser: CardImageParsing {
 private extension CIImage {
 
     func applyNoiseReductionFilter() -> CIImage? {
```

Comment:
No need for `CIImage?`

Reply:
The output image is always an optional image. Applying filters can fail.